beater/beater.go

```diff
@@ -395,6 +397,9 @@ func newServerRunner(ctx context.Context, args serverRunnerParams) (*serverRunne
 	}
 
 	runServerContext, cancel := context.WithCancel(ctx)
+	if args.Namespace == "" {
+		args.Namespace = cfg.DataStreams.Namespace
+	}
```

Comment:
Do you think it would be slightly simpler if we removed this conditional, and instead updated `reloader.reload` to set the initial value of `namespace` to `cfg.DataStreams.Namespace`? Probably not too big of a difference, I was just slightly surprised to find the if/else in this function.